composer.json

```diff
@@ -49,8 +49,8 @@
     "require-dev": {
         "ext-curl": "*",
         "ext-pdo_sqlite": "*",
-        "payum/omnipay-bridge": "^1",
-        "omnipay/dummy": "~2.0",
+        "payum/omnipay-bridge": "^2.0",
+        "omnipay/dummy": "~3.0",
```

Comment:
Not sure but: since omnipay/dummy and payum/omnipay-bridge (not yet) are still in Alpha, we would need to lower the minimum-stability here as well, wouldn't we?